app/facades/food_facade.rb

```diff
@@ -0,0 +1,13 @@
+class FoodFacade
+  def self.search_food(foods)
+    json = FoodService.get_food_data(foods)
+    json[:foods].map do |food|
+      Food.new(food)
+    end
+  end
+
+  def self.total_hits(foods)
+    json = FoodService.get_food_data(foods)
+    json[:totalHits]
+  end
```

Comment:
Following up regarding my comment in your controller, in your `search_food` method, you could return something like 

`[json[:totalHits], foods]` where `foods` is the collection of food objects that you created on lines 4-6.